mne/decoding/_time_frequency_csp.py

```diff
@@ -0,0 +1,186 @@
+# laura gwilliams leg5@nyu.edu
+# run classifier at different time and frequency windows
+
+# Steps:
+# 1) preprocess raw into epochs band-passed at different frequencies and
+#    cropped with X many cycles
+# 2) fit classifier with cross validation
+# 3) predict
+# 4) score
+
+import numpy as np
+
+from mne import Epochs, pick_types, find_events
+from mne.io import concatenate_raws, read_raw_edf
+from mne.datasets import eegbci
+from mne.decoding import CSP
+from mne.filter import band_pass_filter
```

Comment:
all within-module imports should be relative instead of absolute (e.g., `from ..filter import band_pass_filter`

Also `band_pass_filter` is deprecated, you should use `filter_data`